config/runtime.exs

```diff
@@ -11,62 +25,44 @@ database = System.get_env("DB_NAME", "postgres")
+metrics_rpc_timeout_in_ms = Env.get_integer("METRICS_RPC_TIMEOUT_IN_MS", :timer.seconds(15))
+rebalance_check_interval_in_ms = Env.get_integer("REBALANCE_CHECK_INTERVAL_IN_MS", :timer.seconds(10))
+tenant_max_bytes_per_second = Env.get_integer("TENANT_MAX_BYTES_PER_SECOND", 100_000)
+tenant_max_channels_per_client = Env.get_integer("TENANT_MAX_CHANNELS_PER_CLIENT", 100)
+tenant_max_concurrent_users = Env.get_integer("TENANT_MAX_CONCURRENT_USERS", 200)
+tenant_max_events_per_second = Env.get_integer("TENANT_MAX_EVENTS_PER_SECOND", 100)
+tenant_max_joins_per_second = Env.get_integer("TENANT_MAX_JOINS_PER_SECOND", 100)
+rpc_timeout = Env.get_integer("RPC_TIMEOUT", :timer.seconds(30))
+max_gen_rpc_clients = Env.get_integer("MAX_GEN_RPC_CLIENTS", 5)
+run_janitor? = Env.get_boolean("RUN_JANITOR", false)
+janitor_schedule_randomize = Env.get_boolean("JANITOR_SCHEDULE_RANDOMIZE", true)
+janitor_max_children = Env.get_integer("JANITOR_MAX_CHILDREN", 5)
+janitor_chunk_size = Env.get_integer("JANITOR_CHUNK_SIZE", 10)
+janitor_run_after_in_ms = Env.get_integer("JANITOR_RUN_AFTER_IN_MS", :timer.minutes(10))
+janitor_children_timeout = Env.get_integer("JANITOR_CHILDREN_TIMEOUT", :timer.seconds(5))
+janitor_schedule_timer = Env.get_integer("JANITOR_SCHEDULE_TIMER_IN_MS", :timer.hours(4))
 
 disconnect_socket_on_no_channels_interval_in_ms =
-  System.get_env("DISCONNECT_SOCKET_ON_NO_CHANNELS_INTERVAL_IN_MS", "30000") |> String.to_integer()
-
-tenant_max_bytes_per_second = System.get_env("TENANT_MAX_BYTES_PER_SECOND", "100000") |> String.to_integer()
-tenant_max_channels_per_client = System.get_env("TENANT_MAX_CHANNELS_PER_CLIENT", "100") |> String.to_integer()
-tenant_max_concurrent_users = System.get_env("TENANT_MAX_CONCURRENT_USERS", "200") |> String.to_integer()
-tenant_max_events_per_second = System.get_env("TENANT_MAX_EVENTS_PER_SECOND", "100") |> String.to_integer()
-tenant_max_joins_per_second = System.get_env("TENANT_MAX_JOINS_PER_SECOND", "100") |> String.to_integer()
-rpc_timeout = System.get_env("RPC_TIMEOUT", "30000") |> String.to_integer()
-max_gen_rpc_clients = System.get_env("MAX_GEN_RPC_CLIENTS", "5") |> String.to_integer()
-run_janitor? = System.get_env("RUN_JANITOR", "false") == "true"
-janitor_schedule_randomize = System.get_env("JANITOR_SCHEDULE_RANDOMIZE", "true") == "true"
-janitor_max_children = System.get_env("JANITOR_MAX_CHILDREN", "5") |> String.to_integer()
-janitor_chunk_size = System.get_env("JANITOR_CHUNK_SIZE", "10") |> String.to_integer()
-# defaults to 10 minutes
-janitor_run_after_in_ms = System.get_env("JANITOR_RUN_AFTER_IN_MS", "600000") |> String.to_integer()
-# defaults to 5 seconds
-janitor_children_timeout = System.get_env("JANITOR_CHILDREN_TIMEOUT", "5000") |> String.to_integer()
-# 4 hours by default
-janitor_schedule_timer = System.get_env("JANITOR_SCHEDULE_TIMER_IN_MS", "14400000") |> String.to_integer()
-# defaults to 10 minutes
+  Env.get_integer("DISCONNECT_SOCKET_ON_NO_CHANNELS_INTERVAL_IN_MS", :timer.seconds(30))
```

Comment:
I don't think this `disconnect_socket_on_no_channels_interval_in_ms` is being used at all. Should we just remove it?